charts/gateway/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "2.2.1"
+appVersion: "2.2.2"
 description: Conduktor Gateway and associated backing Kafka
 name: conduktor-gateway
 version: 2.2.2
```

Comment:
```suggestion
version: 2.2.3
```

Reply:
https://conduktor-team.slack.com/archives/C04KW5ALNHJ/p1700232376989339

TL;DR needs fixing - https://linear.app/conduktor/issue/PXY-1279/have-helm-released-version-match-the-version-of-gw

Reply:
TLDR; 
```
version: 2.2.4
appVersion: "2.2.2"
```


"appVersion: The version of the app that this contains (optional). Needn't be SemVer. Quotes recommended." https://helm.sh/docs/topics/charts/#the-chartyaml-file
so we try to keep it in sync with image version used by the chart. In this case GW 2.2.2
And we increment the chart `version` every time we make a change respecting semver spec strictly.

Also during the time this PR was on flight, an update to the chart was pushed and merged so now the current chart version of GW is `2.2.3` so **your PR should update it to `2.2.4`** 
Sorry we don't have something on CI/PR that enforce that rule yet so we have to be careful to not override an already released chart version.